pulsar-client-cpp/include/pulsar/Result.h

```diff
@@ -62,8 +62,8 @@ enum Result {
     ResultProducerBlockedQuotaExceededError,     /// Producer is blocked
     ResultProducerBlockedQuotaExceededException,  /// Producer is getting exception
     ResultProducerQueueIsFull, /// Producer queue is full
-
-    ResultMessageTooBig /// Trying to send a messages exceeding the max size
+    ResultMessageTooBig, /// Trying to send a messages exceeding the max size
+    ResultConsumerStatsError /// Broker returned an error while trying to fetch consumer stats
```

Comment:
Cannot we use the existing `ResultError` instead of `ResultConsumerStatsError`. If I am making a call to get consumer stats, there's no need to re-state the type of the request.